README.md

```diff
@@ -40,9 +45,10 @@ Run the following command to ingest all the data.
 python ingest.py
 ```
 
-It will create a `db` folder containing the local vectorstore. Will take time, depending on the size of your documents.
-You can ingest as many documents as you want, and all will be accumulated in the local embeddings database. 
-If you want to start from an empty database, delete the `db` folder.
+If using `Weaviate` this will upsert data to your local vector database instance. If using `Chroma` this will create a `db` folder containing the local vectorstore. Will take time, depending on the size of your documents.
+
+You can ingest as many documents as you want, and all will be accumulated in the local embeddings database.
+If you want to start from an empty database, you can either create a new `Weaviate` instance or if working with `Chroma` you can delete the `db` folder.
```

Comment:
@zainhas about starting with an empty database with weaviate, I think we it's better to mention that by default,  the docker-compose file does not mount PERSISTENCE_DATA_PATH to the host machine's storage so removing and restarting the weaviate container will be the same as starting over with an empty database. That way, if users want to persist the embeddings, they will know what to do.  Also mention another alternative is to use the python client i.e. client.schema.delete_all() instead of restarting.